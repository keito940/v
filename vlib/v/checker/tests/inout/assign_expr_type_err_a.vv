fn main() {
	mut a := 10
	a <<= 0.5
}