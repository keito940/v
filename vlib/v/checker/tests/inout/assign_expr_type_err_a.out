vlib/v/checker/tests/inout/assign_expr_type_err_a.v:3:8: error: operator <<= not defined on right type `f64`
    1| fn main() {
    2|     mut a := 10
    3|     a <<= 0.5
                 ~~~
    4| }
